pkg/features/kube_features.go

```diff
@@ -1052,7 +1053,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	MemoryQoS: {Default: false, PreRelease: featuregate.Alpha},
 
-	MinDomainsInPodTopologySpread: {Default: true, PreRelease: featuregate.Beta},
+	MinDomainsInPodTopologySpread: {Default: true, PreRelease: featuregate.GA},
```

Comment:
As suggested here, it keeps the feature gate itself while removing all the references to this gate.
https://github.com/kubernetes/enhancements/pull/4407#issuecomment-1927256512